app/components/chat/chat.tsx

```
@@ -88,6 +93,15 @@ export function Chat() {
     []
   )
 
+  // Handle transcription text selection
+  const handleTranscriptionSelect = useCallback(
+    (text: string) => {
+      // Set the transcribed text as the input value
+      handleInputChange(text)
+    },
+    [handleInputChange]
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)